app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  validates :rating,
+    presence: true,
+    numericality: { only_integer: true }, inclusion: { in: (1..5) }
+
+  RATING_LIST = [1, 2, 3, 4, 5]
```

Comment:
I really like the use of inner class constants for truthful data. 

Traditionally, these are kept at the top of a class ( inside the class definition ), but If you find it belongs elsewhere in the class (or in any other class), go with your judgement!